### ISSUE TITLE
Q: `ChrootFileSystem.isWatchSupported`

### ISSUE THREAD

**Issue by matanlurey** [matanlurey/file#41]
Is this currently `false` out of convenience?

Assuming the underlying FS already supports it, how much work is it?

**Comment by tvolkert** [comment]
Yep, it's currently not implemented out of convenience.  It shouldn't be too much work.
